pages/snowboard/triggers.htm

```diff
@@ -0,0 +1,279 @@
+title = "Triggers | Snowboard"
+url = "/snowboard/triggers"
+layout = "snowboard"
+==
+<section id="body" class="py-20 flex-grow">
+    <div class="container max-w-screen-xl">
+        <div class="block lg:flex lg:flex-row divide-x divide-gray-100">
+            <div class="flew-grow-0 px-8 lg:w-2/12 xl:pr-0">
```

Comment:
```suggestion
            <div class="flex-grow-0 px-8 lg:w-2/12 xl:pr-0">
```

Reply:
Wow, found a lot of mispellings lol. Thanks @mjauvin 😂